scripts/generator/graphql_generator/csharp/SDK/CartLineItems.cs.erb

```diff
@@ -147,7 +155,7 @@ namespace <%= namespace %>.SDK {
         ///
         /// cart.LineItems.AddOrUpdate(variantId, 3);
         /// \endcode
-        public void AddOrUpdate(string variantId, long? quantity = null, Dictionary<string, string> customAttributes = null) {
+        public void AddOrUpdate(string variantId, decimal price, long? quantity = null, Dictionary<string, string> customAttributes = null) {
             CartLineItem input = Get(variantId);
 
             if (input != null) {
```

Comment:
Under this case when the cart line item already exists, should price be updated as well?